presto-main/src/test/java/com/facebook/presto/cost/TestCostCalculator.java

```diff
@@ -746,7 +748,11 @@ private TableScanNode tableScan(String id, String... symbols)
         TpchTableHandle tableHandle = new TpchTableHandle("orders", 1.0);
         return new TableScanNode(
                 new PlanNodeId(id),
-                new TableHandle(new ConnectorId("tpch"), new TpchTableHandle("orders", 1.0)),
+                new TableHandle(
+                        new ConnectorId("tpch"),
+                        new TpchTableHandle("orders", 1.0),
+                        TestingTransactionHandle.create(),
```

Comment:
nit: Why not use `TpchTransactionHandle.INSTANCE`, this is consistent with line 758.